.github/workflows/test-build.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+
+name: Event Pro CI

```

Comment:
The name of the workflow might make more sense to be something like "Build for Test Environment" or "Build for Test Env"